TLM/TLM/Patch/_VehicleAI/UpdatePathTargetPositionsPatch.cs

```diff
@@ -719,10 +720,10 @@ public class UpdatePathTargetPositionsPatch {
                 // calculate next segment offset
                 byte nextSegOffset;
                 if ((vehicleData.m_flags & Vehicle.Flags.Flying) != 0) {
-                    Log._DebugIf(
-                        logLogic,
-                        () => $"CustomVehicle.CustomUpdatePathTargetPositions({vehicleID}): " +
-                        "Vehicle is flying");
+                    if (logLogic) {
```

Comment:
if (logLogic) should not be necessary due to [Conditional("DEBUG")]

Reply:
I'll try but I think I've changed everything because I wasn't sure where is the source and also which one is triggering generation of helper classes, since just one occurrence is enough and there are +20 XD